src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -142,6 +143,10 @@ public sealed partial class Server : IDisposable {
                 TypeshedPath = @params.initializationOptions.typeStubSearchPaths.FirstOrDefault()
             };
 
+            if (@params.initializationOptions.enableAnalysCache != false) {
+                //_services.AddService(new ModuleDatabase(_services));
```

Comment:
I think we can uncomment this in the branch

Reply:
Done